pypesto/optimize/optimizer.py

```diff
@@ -41,6 +41,16 @@ def __init__(self, optimizer: str):
         )
 
 
+def add_inner_parameters(
```

Comment:
```suggestion
def _add_inner_parameters(
```

Not meant to be used elsewhere, right?